.orchestra/ci/ci-run.sh

```diff
@@ -250,7 +250,18 @@ if test "$PUSH_CHANGES" = 1; then
         git add .
 
         if ! git diff --cached --quiet; then
-            git commit -m'Automatic binary archives'
+            set +x
+            COMMIT_MSG=$(cat <<EOF
+Automatic binary archives
+
+ORCHESTRA_CONFIG_COMMIT=$(git -C "$ORCHESTRA_ROOT" rev-parse --short HEAD || true)
+ORCHESTRA_CONFIG_BRANCH=$(git -C "$ORCHESTRA_ROOT" name-rev --name-only HEAD || true)
+COMPONENT_TARGET_BRANCH=$COMPONENT_TARGET_BRANCH
+EOF
+)
```

Comment:
```suggestion
            COMMIT_MSG="Automatic binary archives

ORCHESTRA_CONFIG_COMMIT=$(git -C "$ORCHESTRA_ROOT" rev-parse --short HEAD || true)
ORCHESTRA_CONFIG_BRANCH=$(git -C "$ORCHESTRA_ROOT" name-rev --name-only HEAD || true)
COMPONENT_TARGET_BRANCH=$COMPONENT_TARGET_BRANCH"
```

I'm fixing this. No action required from your side.